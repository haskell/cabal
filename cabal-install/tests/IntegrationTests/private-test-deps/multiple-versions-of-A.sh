. ./common.sh

# Create the sandbox
cabal sandbox init >/dev/null

# Add additional sources
cabal sandbox add-source deps/A-1 >/dev/null
cabal sandbox add-source deps/T   >/dev/null

# Install 
cabal install --enable-tests >/dev/null

# Run the test
# We don't know the name of the sandbox dir,
# but there will only be one so we can use '*'
dist/*/build/T-test/T-test
