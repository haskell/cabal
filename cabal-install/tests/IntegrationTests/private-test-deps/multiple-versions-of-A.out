([("A",2)],[("T",1),("A",1)])
